ui/src/app/views/workspace-edit-underserved-population/component.tsx

```diff
@@ -0,0 +1,321 @@
+import {Clickable} from 'app/components/buttons';
```

Comment:
This should probably go into `ui/src/app/components`, rather than `ui/src/app/views`.

Reply:
Question: So does component contains classes/modals that are used by multiple pages like button /alerts etc?
If no i can move this to component. However since it is a part of Workspace-Edit and will be used only by that , i prefer to see it in views.

Reply:
I am sort of leaning towards having views just be top level pages, and then sub-components either live in the page file, but this seems fine for now until we can have that conversation for sure.